hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -394,6 +403,8 @@ private OzoneManager(OzoneConfiguration conf) throws IOException,
         OMConfigKeys.OZONE_OM_RATIS_ENABLE_KEY,
         OMConfigKeys.OZONE_OM_RATIS_ENABLE_DEFAULT);
 
+    omEpoch = OmUtils.getOMEpoch(isRatisEnabled);
```

Comment:
We can reuse metadataManager#getOmEpoch to set epoch value.

Reply:
Done.